app/code/Magento/Newsletter/Observer/PredispatchNewsletterObserver.php

```diff
@@ -12,16 +12,18 @@
 use Magento\Framework\Event\ObserverInterface;
 use Magento\Framework\UrlInterface;
 use Magento\Store\Model\ScopeInterface;
+use Magento\Newsletter\Model\Config;
+use Magento\Framework\App\ObjectManager;
 
 /**
  * Class PredispatchNewsletterObserver
  */
 class PredispatchNewsletterObserver implements ObserverInterface
 {
     /**
-     * Configuration path to newsletter active setting
+     * @var Newsletter Config
      */
-    const XML_PATH_NEWSLETTER_ACTIVE = 'newsletter/general/active';
```

Comment:
Please mark as deprecated and add `use Magento\Newsletter\Model\Config::XML_PATH_NEWSLETTER_ACTIVE instead` comment.